src/modules/feature-modules/innovator/pages/innovation/manage-access/manage-access-leave-innovation.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CoreComponent } from '@app/base';
+import { InnovationsService } from '@modules/shared/services/innovations.service';
+import { ContextInnovationType } from '@modules/stores';
+import { InnovationCollaboratorStatusEnum } from '@modules/stores/innovation/innovation.enums';
+
+@Component({
+  selector: 'app-innovator-pages-innovation-manage-access-leave-innovation',
+  templateUrl: './manage-access-leave-innovation.component.html'
+})
+export class PageInnovationManageAccessLeaveInnovationComponent extends CoreComponent implements OnInit {
+
+  innovationId: string;
+  innovation: ContextInnovationType;
+
+  submitButton = { isActive: true, label: 'Leave innovation' };
+
+  constructor(
+    private activatedRoute: ActivatedRoute,
+    private innovationsService: InnovationsService
+  ) {
+
+    super();
+
+    this.innovationId = this.activatedRoute.snapshot.params.innovationId;
+    this.innovation = this.stores.context.getInnovation();
+
+    this.setPageTitle(`Leave "${this.innovation.name}" innovation`);
```

Comment:
it needs be with ' instead of "

Reply:
Changed double quotation to single quotation